presto-main/src/main/java/com/facebook/presto/operator/TableWriterOperator.java

```diff
@@ -286,11 +296,14 @@ private Page createStatisticsPage(Page aggregationOutput)
         int positionCount = aggregationOutput.getPositionCount();
         Block[] outputBlocks = new Block[types.size()];
         for (int channel = 0; channel < types.size(); channel++) {
-            if (channel < STATS_START_CHANNEL) {
+            if (channel == CONTEXT_CHANNEL) {
```

Comment:
nit: I would put this if nested in `(channel < STATS_START_CHANNEL)`. And add a comment saying we have operator context (although identical) for statistic pages.